src/style.css

```diff
@@ -0,0 +1,149 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+}
+
+/* #region card__img */
+
+.card__img {
+
+  object-fit: cover;
+  object-position: center;
+  left: 0;
+  right: 0;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+
+}
+
+/* #endregion */
+
+/* #region product-name */
+
+.card_product-name {
+  display: block;
+  width: 166px;
+  height: 36px;
+  font-weight: 500;
+  font-size: 12px;
+  margin-top: 0;
+  margin-bottom: 4px;
+  line-height: 18px;
+  color: #060b35;
+
+}
+
+.card_product-code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin-bottom: 17px;
+  margin-top: 0;
+}
+
+/* #endregion */
+
+/* #region stars */
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars--4 > .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars__reviews-amoumt {
+  color: #060b35;
+  font-weight: 400;
+  font-size: 10px;
+  margin: 0;
+}
+
+/* #endregion */
+
+/* #region price */
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+  margin-top: 24px;
+
+}
+
+.card__price_title {
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+  width: 31px;
+}
+
+.card__price_price-amoumt {
+  display: block;
+  color: #060b35;
+  font-weight: 700;
+  font-size: 16px;
+  width: 50px;
+  height: 18px;
+  line-height: 18px;
+}
+/* #endregion */
+
+/* #region button-buy */
+
+.card__buy-button {
+  display: flex;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  background: #00acdc;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  text-decoration: none;
+}
+
+.card__buy-button:hover {
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  background: #fff;
+  color: #00acdc;
+  border: solid #00acdc;
```

Comment:
checklist 9, don't add new border to the element on hover. Add default
transparent border of the same width, and change its color on `:hover`